doc/developer-documentation/aeproject-cli/tx-inspect.md

```diff
@@ -0,0 +1,43 @@
+# aeproject inspect
+
+## Inspect raw tx 
+The **inspect** command unpack and verify raw transaction (verify nonce, ttl, fee, account balance).
+```text
+    aeproject inspect -tx tx_+E0MAaEBK4bq9XiZ/0QVdOa8Hs9V18v6dGZYIa8XXNYFpQh6yq6hAR8To7CL8AFABmKmi2nYdfeAPOxMCGR/btXYTHiXvVCjCoJOIAABgL6cZFo=
```

Comment:
do we need the **-tx** we have only tx inspect so I find this parameter not needed